<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UT343D V1.02</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_OpenSerial">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>78</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>78</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>78</width>
      <height>51</height>
     </size>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_CloseSerial">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>78</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>78</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>78</width>
      <height>51</height>
     </size>
    </property>
    <property name="text">
     <string>关闭串口</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>51</y>
      <width>966</width>
      <height>492</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="minimumSize">
          <size>
           <width>186</width>
           <height>117</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>185</width>
           <height>117</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeView">
         <property name="minimumSize">
          <size>
           <width>185</width>
           <height>339</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>185</width>
           <height>339</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>770</width>
         <height>490</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>770</width>
         <height>490</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUTD343">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionStart"/>
    <addaction name="actionTop"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionClose_2"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="actionLanguage"/>
   </widget>
   <addaction name="menuUTD343"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionTop">
   <property name="text">
    <string>STop</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionClose_2">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionLanguage">
   <property name="text">
    <string>Language</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
